toml.md

```diff
@@ -545,6 +544,10 @@ sf5 = +nan # same as `nan`
 sf6 = -nan # valid, actual encoding is implementation-specific
 ```
 
+Implementations are free to support any precision level. It's recommended that
+at least IEEE 754 binary64 values are accepted and handled losslessly. If a
+floating point cannot be represented losslessly, an error must be thrown.
+
```

Comment:
```suggestion
Implementations are free to support any precision level. It's recommended that
at least IEEE 754 binary64 values are accepted and stored without losing bits.
If a floating point cannot be accepted and stored, an error must be thrown.
```

Reply:
(an attempt at addressing concerns from the review above)

Reply:
@pradyunsg Not sure that'll help – what does "losing bits" means? In the file, floats are actually stored as decimals, so there is no clear "bit" equivalent, and conversion into a float representation will in most cases be slightly lossy. Also, how to decide which values "cannot be accepted and stored"?